packaging/windows/Resources/LICENSE.rtf

```diff
@@ -0,0 +1,194 @@
+\line       Notwithstanding the above, nothing herein shall supersede or modify
+\line       the terms of any separate license agreement you may have executed
+\line       with Licensor regarding such Contributions.
+\line 
+\line    6. Trademarks. This License does not grant permission to use the trade
+\line       names, trademarks, service marks, or product names of the Licensor,
+\line       except as required for reasonable and customary use in describing the
+\line       origin of the Work and reproducing the content of the NOTICE file.
+\line 
+\line    7. Disclaimer of Warranty. Unless required by applicable law or
+\line       agreed to in writing, Licensor provides the Work (and each
+\line       Contributor provides its Contributions) on an "AS IS" BASIS,
+\line       WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+\line       implied, including, without limitation, any warranties or conditions
+\line       of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A
+\line       PARTICULAR PURPOSE. You are solely responsible for determining the
+\line       appropriateness of using or redistributing the Work and assume any
+\line       risks associated with Your exercise of permissions under this License.
+\line 
+\line    8. Limitation of Liability. In no event and under no legal theory,
+\line       whether in tort (including negligence), contract, or otherwise,
+\line       unless required by applicable law (such as deliberate and grossly
+\line       negligent acts) or agreed to in writing, shall any Contributor be
+\line       liable to You for damages, including any direct, indirect, special,
+\line       incidental, or consequential damages of any character arising as a
+\line       result of this License or out of the use or inability to use the
+\line       Work (including but not limited to damages for loss of goodwill,
+\line       work stoppage, computer failure or malfunction, or any and all
+\line       other commercial damages or losses), even if such Contributor
+\line       has been advised of the possibility of such damages.
+\line 
+\line    9. Accepting Warranty or Additional Liability. While redistributing
+\line       the Work or Derivative Works thereof, You may choose to offer,
+\line       and charge a fee for, acceptance of support, warranty, indemnity,
+\line       or other liability obligations and/or rights consistent with this
+\line       License. However, in accepting such obligations, You may act only
+\line       on Your own behalf and on Your sole responsibility, not on behalf
+\line       of any other Contributor, and only if You agree to indemnify,
+\line       defend, and hold each Contributor harmless for any liability
+\line       incurred by, or claims asserted against, such Contributor by reason
+\line       of your accepting any such warranty or additional liability.
+\line 
+\line    END OF TERMS AND CONDITIONS
+\line 
+\line    Copyright 2019 Red Hat, Inc.

```

Comment:
We could update it to 2021, the format is supposed to be like:
```
Copyright [yyyy] [name of copyright owner]
``` 

but does this also mean we update it every year, or just the year when we added the license?

Reply:
I never remember when to updated it. However, in this case, we could drop everything starting from "END OF TERMS AND CONDITIONS" (included?)
http://www.apache.org/licenses/LICENSE-2.0.html says:
> To apply the Apache License to specific files in your work, attach the following boilerplate declaration, with the fields enclosed by brackets "[]" replaced with your own identifying information. 

the boilerplate being the part of the license file we are currently discussing.
Since this is the license of the whole work we are showing, not the license of a specific file, I think we can skip it.

Reply:
Right, removed the end bit